src/icf/genCairo.ts

```diff
@@ -55,7 +55,7 @@ function getStarknetLangDirective(): string[] {
 
 function getForwarderInterface(abi: AbiType): string[] {
   return [
-    '@contract_interface',
+    '#[abi]',
     'namespace Forwarder {',
```

Comment:
Why is the change needed? The code still won't work since the rest of the code is cairo 0, right? Or am I missing something? Is there a `namespace` in cairo 1?

Reply:
it must changed because of the global regex replace, yes it doesn't work now